<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameControllerInfoWidget</class>
 <widget class="QWidget" name="GameControllerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Device:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="deviceNameLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Local Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="devicePathLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Force Effect Slots:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="forceEffectSlotsLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device Capabilities</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="canRenderConstant">
        <property name="text">
         <string>Can render CONSTANT force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderPeriodic">
        <property name="text">
         <string>Can render PERIODIC force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderSquare">
        <property name="text">
         <string>Can render SQUARE force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderTriangle">
        <property name="text">
         <string>Can render TRIANGLE force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderSine">
        <property name="text">
         <string>Can render SINE force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderSawUp">
        <property name="text">
         <string>Can render SAW_UP force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderSawDown">
        <property name="text">
         <string>Can render SAW_DOWN force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderCustom">
        <property name="text">
         <string>Can render CUSTOM force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderRamp">
        <property name="text">
         <string>Can render RAMP force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderSpring">
        <property name="text">
         <string>Can render SPRING force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderFriction">
        <property name="text">
         <string>Can render FRICTION force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderDamper">
        <property name="text">
         <string>Can render DAMPER force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderRumble">
        <property name="text">
         <string>Can render RUMBLE force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canRenderInertia">
        <property name="text">
         <string>Can render INERTIA force.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canAdjustGain">
        <property name="text">
         <string>Can adjust GAIN.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="canAdjustAutocenter">
        <property name="text">
         <string>Can adjust AUTOCENTER.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
